filesystem_test.go

```diff
@@ -325,6 +326,79 @@ func TestFilesystemFetchContainerConfigFailingPodIDEmpty(t *testing.T) {
 	}
 }
 
+func TestFilesystemFetchContainerMountsSuccessful(t *testing.T) {
+	fs := &filesystem{}
+	contID := "100"
+
+	contMountsDir := filepath.Join(runStoragePath, testPodID, contID)
+	os.MkdirAll(contMountsDir, dirMode)
+
+	path := filepath.Join(contMountsDir, mountsFile)
+	os.Remove(path)
+
+	t.Logf("Path: %s", path)
+	f, err := os.Create(path)
+	if err != nil {
+		t.Fatal(err)
+	}
+
+	source := "/dev/sda1"
+	dest := "/root"
+	mntType := "ext4"
+	options := "rw"
+	ignoreMount := false
+	hostPath := "/tmp/root"
+
+	mountData := fmt.Sprintf("[{\"Source\":\"%s\",\"Destination\":\"%s\",\"Type\":\"%s\",\"Options\": [\"%s\"],\"IgnoreMount\":%s, \"HostPath\":\"%s\"}]", source, dest, mntType, options, strconv.FormatBool(ignoreMount), hostPath)
```

Comment:
This would look better and be easier to read if you used backticks:

```
mountData := fmt.Sprintf(`
  [
     {
        "Source": "%s",
  ...
`, source, ...)